qa/os/src/test/java/org/elasticsearch/packaging/util/Packages.java

```diff
@@ -286,7 +286,7 @@ private static void verifyDefaultInstallation(Installation es) {
      */
     public static void clearJournal(Shell sh) {
         if (isSystemd()) {
-            sh.run("rm -rf /run/log/journal/*");
+            sh.run("rm -rf /run/log/journal/* /var/log/journal/*");
```

Comment:
@rjernst All refactoring aside, here's the one-line fix for the test failure. Does this look OK to you?

Reply:
Yes that seems fine.